project_national/fact_sheets_category_13.yml

```diff
@@ -0,0 +1,1526 @@
+      - *water_heater_efficiency_electric_heat_pump_66_gal_3_pt_35_uef
+      - *water_heater_efficiency_electric_heat_pump_80_gal_3_pt_45_uef
+
+  - upgrade_name: HPWH + EnergyStar cold climate ductless MSHP w/ existing propane backup
+  # Category 4, Measure 4 + Category 6, Measure 5
+    package_apply_logic: &dualfuel_MSHP_apply_logic_propane
+      - HVAC Has Ducts|No
+      - Heating Fuel|Natural Gas
+      - not:
+          or:
+            - HVAC Heating Efficiency|Shared Heating
+            - HVAC Cooling Efficiency|Shared Cooling  
+    options:
+      - *ductless_MSHP_EnergyStarCC_exist_backup
+
+      - *HP_cooling_efficiency_nonducted
+      - *100_percent_conditioned
+
+      - *propane_boiler_secondary_heating_type_fuel
+      - *secondary_boiler_76
+      - *secondary_boiler_80
+      - *secondary_boiler_90
+      - *propane_wallfloor_furnace_secondary_heating_type_fuel
+      - *secondary_wallfloor_furnace_60
+      - *secondary_wallfloor_furnace_68
+
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0     
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+      - *water_heater_fuel_type
+      - *water_heater_efficiency_electric_heat_pump_50_gal_3_pt_45_uef
+      - *water_heater_efficiency_electric_heat_pump_66_gal_3_pt_35_uef
+      - *water_heater_efficiency_electric_heat_pump_80_gal_3_pt_45_uef
+
+  - upgrade_name: HPWH + EnergyStar cold climate ductless MSHP w/ existing fuel oil backup
+  # Category 4, Measure 4 + Category 6, Measure 5
+    package_apply_logic: &dualfuel_MSHP_apply_logic_FO
+      - HVAC Has Ducts|No
+      - Heating Fuel|Natural Gas
+      - not:
+          or:
+            - HVAC Heating Efficiency|Shared Heating
+            - HVAC Cooling Efficiency|Shared Cooling  
+    options:
+      - *ductless_MSHP_EnergyStarCC_exist_backup
+
+      - *HP_cooling_efficiency_nonducted
+      - *100_percent_conditioned
+
+      - *fuel_oil_boiler_secondary_heating_type_fuel
+      - *secondary_boiler_76
+      - *secondary_boiler_80
+      - *secondary_boiler_90
+      - *fuel_oil_wallfloor_furnace_secondary_heating_type_fuel
+      - *secondary_wallfloor_furnace_60
+      - *secondary_wallfloor_furnace_68
+
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0     
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+      - *water_heater_fuel_type
+      - *water_heater_efficiency_electric_heat_pump_50_gal_3_pt_45_uef
+      - *water_heater_efficiency_electric_heat_pump_66_gal_3_pt_35_uef
+      - *water_heater_efficiency_electric_heat_pump_80_gal_3_pt_45_uef
+
+  - upgrade_name: HPWH + Light Touch Envelope
+  # Category 4, Measure 4 + Category 2, Measure 1
```

Comment:
Do we want to have package_apply_logic here to limit applicability to those that can be upgraded to HPWH only? Same for other HPWH + Envelope packages. @whiphi92 @ekpresent